app/constants/index.ts

```diff
@@ -62,13 +62,15 @@ export const REVOKE_DELEGATION_TX_SIZE_BYTES = 165;
 
 export const STACKING_CONTRACT_CALL_FEE = 260;
 
-export const POOLED_STACKING_TX_SIZE_BYTES = 199;
+export const POOLED_STACKING_TX_SIZE_BYTES = 216;
```

Comment:
this never changes?

Reply:
So, I think I just got it wrong the first time. But could be proven wrong, if so then this implementation needs to change.